src/Type/Accessory/AccessoryLiteralStringType.php

```diff
@@ -171,6 +171,11 @@ public function toNumber(): Type
 		return new ErrorType();
 	}
 
+	public function toAbsoluteNumber(): Type
+	{
+		return new ErrorType();
```

Comment:
you should add a test for `abs('123')` and `abs('hello')` and fix this type

Reply:
I added a few tests but they passed without any code change.

Reply:
I mixed it up. thanks for adding the tests. it seems returning the ErrorType here is correct (and in the other comments)